--- a/src/CMakeLists.txt.orig	2025-04-01 15:45:48.135103010 +0000
+++ b/src/CMakeLists.txt	2025-04-01 15:46:31.890720003 +0000
@@ -63,13 +63,3 @@
 install(FILES ${includes} DESTINATION include/chiabls)
 install(FILES ${C_LIB} DESTINATION lib)
 
-add_executable(runtest test.cpp)
-add_executable(runbench test-bench.cpp)
-
-if (SODIUM_FOUND)
-  target_link_libraries(runtest chiablstmp relic_s sodium)
-  target_link_libraries(runbench chiablstmp relic_s sodium)
-else()
-  target_link_libraries(runtest chiablstmp relic_s)
-  target_link_libraries(runbench chiablstmp relic_s)
-endif()
